packages/opentelemetry-cloud-monitoring-exporter/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../tsconfig.base",
+  "extends": "./node_modules/gts/tsconfig-google.json",
```

Comment:
The diff of `./node_modules/gts/tsconfig-google.json` vs `..tsconfig.base`:

```diff
6a7
>     "lib": ["es2018"],
11d11
<     "noUnusedLocals": true,
15,17c15
<     "strictNullChecks": true,
<     "target": "es2017",
<     "incremental": true
---
>     "target": "es2018"
```

- `strictNullChecks` is not changed; they both have `strict: true` which covers this
- `incremental` is not changed; [`incremental` is true if `composite`](https://www.typescriptlang.org/tsconfig#incremental) and [`composite` defaults to true](https://www.typescriptlang.org/tsconfig#composite).
- The only difference is target/lib. [`es2018` is for node 10 and higher](https://www.typescriptlang.org/tsconfig#target)